recipe/meta.yaml

```diff
@@ -1,25 +1,27 @@
 {% set name = "cryptography-vectors" %}
-{% set version = "41.0.7" %}
+{% set version = "42.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cryptography_vectors-{{ version }}.tar.gz
-  sha256: 7b36f976b6e58cc1801310e1c93c584c6539d371da7f8538edd8fc463dc80d5b
+  sha256: adcdccf5d9ee661a9602ad21d2525f678ba07a6e768ce79835994e208bab0e16
 
 build:
-  skip: true  # [py<37]
+  skip: true  # [py<38]
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
 
 requirements:
   host:
     - python
     - pip
+    - flit-core
     - setuptools
     - wheel
```

Comment:
```suggestion
```
`flit-core` replaces `setuptools` and `wheel` in this case.

Reply:
I actually asked Ryan about this because I knew that `flit-core` was just `python`, `pip`, `setuptools`, and `wheel`, but he said it was something different, so I added them back for explicitness.